modules/programs/nix-your-shell.nix

```
@@ -0,0 +1,56 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.nix-your-shell;
+
+in {
+  meta.maintainers = [ maintainers.terlar ];
+
+  options.programs.nix-your-shell = {
+    enable = mkEnableOption ''
+      {command}`nix-your-shell`, a wrapper for `nix develop` or `nix-shell`
+      to retain the same shell inside the new environment.
+    '';
```

the trailing period is added by `mkEnableOption`

```suggestion
      to retain the same shell inside the new environment'';
```

---

putting `''` on the same line was intentional (a space is added before the `.` otherwise)

---

Sorry, missed that, pushed a fix.